driver/driver.go

```diff
@@ -17,16 +17,11 @@ type Driver interface {
 	Stats(api.ScannerStats) error
 }
 
-func Run(out *MuxOutput, d Driver, statsInterval time.Duration) error {
+func Run(out *MuxOutput, d Driver, statsTicker <-chan time.Time) error {
```

Comment:
Nit: This name reads like it's a time.Ticker, which it isn't. I think statsTicks or statsTickCh would be a better name.

Reply:
https://github.com/brimsec/zq/pull/669